lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -18,6 +18,8 @@ defmodule Membrane.H264.FFmpeg.Encoder do
   alias Membrane.H264.FFmpeg.Common
   alias Membrane.RawVideo
 
+  @no_pts -9_223_372_036_854_775_808
```

Comment:
EDIT: This will be invalid after the next suggestion, I'm keeping it for educational purposes :wink:  

Instead of copy-pasting the value, I'd refer to the value in Common (it should be exposed via function):
```suggestion
  @no_pts Common.no_pts()
```